src/App.js

```diff
@@ -1,104 +1,44 @@
 import React, { useState, useEffect } from "react";
+import AddTodoForm from "./AddTodoForm";
+import TodoList from "./TodoList";
 
-function App() {
+const App = () => {
   const [todoList, setTodoList] = useState([]);
-  const [newTodo, setNewTodo] = useState("");
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
-    const fetchData = new Promise((resolve) => {
-      setTimeout(() => {
-        resolve({ data: { todoList: [] } });
-      }, 2000);
-    });
-
-    fetchData.then((result) => {
-      setTodoList(result.data.todoList);
-      setIsLoading(false);
-    });
+    const fetchData = () => {
+      new Promise((resolve) => {
+        setTimeout(() => {
+          resolve({ data: { todoList: [] } });
```

Comment:
At this point you need to get the todo list from the localstorage, please fix. if you are confused please set a meeting with me